### PR TITLE
Show Celo Default Recipient only when devMode is active

### LINKED ISSUE #8: Users SNBAT see Celo Default Recipient
### Expected Behavior

Celo Default Recipient should not be shown

### Current Behavior

Celo Default Recipient is listed for testing

### PR DESCRIPTION
**Pull request by drklee3**
### Description

Hides the Celo Default Recipient to users, but shows it when devMode is active for testing purposes (same as the DevSkipButtons).  Adds a new `No contacts found` message when the RecipientPicker is empty instead of a `No results for ""` message.

### Tested

Tested with devMode on and off to ensure the Celo Default Recipient displays correctly.

| devModeActive == true | devModeActive == false |
| --------------------- | ---------------------- |
| devModeActive && empty | !devModeActive && empty
| devModeActive && filled | !devModeActive && filled |

Tests also pass after updating the Invite Screen snapshot.

### Other changes

N/A

### Related issues

- Fixes #8

### CODE REVIEW

**Comment by jmrossy** [comment]
I reworked our handling of recipients (especially recent recipients) in this PR here:
https://github.com/celo-org/celo-monorepo/pull/396

I should have that merged today, sorry for the conflicts it'll make here!

**Comment by drklee3** [comment]
> I reworked our handling of recipients (especially recent recipients) in this PR here:
> #396
> 
> I should have that merged today, sorry for the conflicts it'll make here!

No worries! I should have merged master in and re-tested now.

File: packages/mobile/src/recipients/RecipientPicker.tsx [hunk 1]
```diff
@@ -137,10 +137,16 @@ export class RecipientPicker extends React.Component<RecipientProps> {
...
+            <Text style={[fontStyles.body, style.emptyViewBodyDark]}>
+              {` "${this.props.searchQuery}"`}
+            </Text>
+          </>
+        ) : (
+          <Text style={fontStyles.body}>{this.props.t('noContacts')}</Text>
+        )}
       </View>
       <Text style={[fontStyles.bodySmall, style.emptyViewBodySmall]}>
         {this.props.t('searchForSomeone')}
```
Comment (jmrossy) [inline packages/mobile/src/recipients/RecipientPicker.tsx]: This shouldn't show if there are no contacts.

**Review by jmrossy — COMMENTED** [review]
See comment below about the no contacts string.
Also, can you align the font stylings between your left and right screenshots? Left's use the left style for both.

**Review by ashishb — APPROVED** [review]

**Pull request merged**